src/ol/format/igc.js

```diff
@@ -202,3 +202,24 @@ ol.format.IGC.prototype.readFeaturesFromText = function(text, opt_options) {
  * @api
  */
 ol.format.IGC.prototype.readProjection;
+
+
+/**
+ * Not implemented.
+ * @inheritDoc
+ */
+ol.format.IGC.prototype.writeFeatureText = function(feature, opt_options) {};
```

Comment:
Any reason that one time

```
/**
 * Not implemented.
 * @inheritDoc
 */
ol.format.IGC.prototype.writeFeatureText = function(feature, opt_options) {};
```
... is used, and another time
```
/**
 * @override
 */
ol.format.IGC.prototype.readGeometryFromText = function() {
  throw new Error('Not implemented');
};
```

What's the difference?